sdk/cosmos/azure-cosmos-benchmark/ctl/run_benchmark.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+serviceEndpoint=$Endpoint
```

Comment:
please follow bash code style for naming variables
https://google.github.io/styleguide/shellguide.html#s7-naming-conventions

Reply:
Yes left hand side we can do i.e. serviceEndpoint -> service_endpoint . However right hand side are docker env  variables which does not have as such naming convention mention on [docker docs](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file). Also we already have contract established with CTL on those. So If you have strong opinion on docker env  variables please let me know

Reply:
Naveen I am referring to all variable names in this file. we should follow the bash code style.

Reply:
I know , i was trying to avoid those convention on env variable from docker, as it involve updating scripts on ctl . Anyway I did that now , will contact ctl team